content/en/Platform Deep Dive/Pentests/pentest-types.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Comprehensive Pentests"
+linkTitle: "Comprehensive Pentests"
+weight: 5
+description: >
+  Learn more about Comprehensive Pentests that we offer.
+---
+
+{{% pageinfo %}}
+We're working to introduce more pentest types soon. Stay tuned.
+{{% /pageinfo %}}
+
+You can launch Comprehensive Pentests on our [PtaaS](/getting-started/glossary/#pentest-as-a-service-ptaas) platform. Refer to the table below for details.
+
+| | Comprehensive Pentest |
+| --- | --- |
+| **Definition** | {{% comprehensive-pentest %}} |
+| **Credit Requirements** | <li>5 credits minimum, thereafter in increments of 1 credit</li><li>{{% pentest-big-scope %}}</li> |
+| **Pentest Scope** | Broad area of an [asset](/getting-started/glossary/#asset)<!--<br><li>The pentest is focused on breadth</li>--> |
+| **Use Cases** | <li>Comprehensive [security audit](/getting-started/glossary/#security-audit)</li><li>[Compliance audit](/getting-started/glossary/#compliance-audit) testing based on the frameworks such as [SOC 2](https://us.aicpa.org/interestareas/frc/assuranceadvisoryservices/aicpasoc2report.html), [ISO 27001](https://www.iso.org/isoiec-27001-information-security.html), [PCI-DSS](https://www.pcisecuritystandards.org/), [CREST](https://www.crest-approved.org/), or [HIPAA](https://www.hhs.gov/hipaa/index.html)</li><li>M&amp;A due diligence</li><li>Internal or third-party attestation request</li> |
```

Comment:
What's "M&A"? If we're talking about pentests of combined companies, I assume we're talking about "Mergers and Acquisitions"

```suggestion
| **Use Cases** | <li>Comprehensive [security audit](/getting-started/glossary/#security-audit)</li><li>[Compliance audit](/getting-started/glossary/#compliance-audit) testing based on the frameworks such as [SOC 2](https://us.aicpa.org/interestareas/frc/assuranceadvisoryservices/aicpasoc2report.html), [ISO 27001](https://www.iso.org/isoiec-27001-information-security.html), [PCI-DSS](https://www.pcisecuritystandards.org/), [CREST](https://www.crest-approved.org/), or [HIPAA](https://www.hhs.gov/hipaa/index.html)</li><li>Mergers and Acquisitions due diligence</li><li>Internal or third-party attestation request</li> |
```